> error NU1102: Unable to find package Microsoft.Extensions.Http.Resilience with version (>= 8.1.0-preview.23565.2) [D:\a\_work\1\s\.dotnet\sdk\8.0.100\NuGet.targets]

Hmm, this built on my machine, but I had the package already cached. I guess this feed is needed.

This error does touch on the servicing conversation though. Are we guaranteed that `Microsoft.Extensions.Http.Resilience` version `8.1.0` will ship on nuget.org before we ship our preview2? cc @joperezr

--- a/NuGet.config
+++ b/NuGet.config
@@ -6,10 +6,14 @@
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
   </packageSources>
   <packageSourceMapping>
     <packageSource key="dotnet-public">
+      <package pattern="*" />
+    </packageSource>
+    <packageSource key="dotnet8">
       <package pattern="*" />
     </packageSource>
     <packageSource key="dotnet-libraries">

